src/TabBar.js

```diff
@@ -428,13 +428,16 @@ export default class TabBar extends PureComponent<DefaultProps, Props, State> {
                 tabContainerStyle.flex = 1;
               }
 
+              const accessible = route.accessible || true;
```

Comment:
This looks wrong. It'll always ensure that `accessible` is `true`, since it assigns the variable when `route.accessible` is true, otherwise sets it to true.